doc/manual/de/install/allgemein.rst

```diff
@@ -62,6 +62,12 @@ MQTT
 Damit die CometVisu ein MQTT-Teilnehmer ist, wird ein MQTT Broker benötigt
 der über WebSocket ansprechbar ist.
 
+ioBroker
+~~~~~~~~
+
+Die Installation des "Web socket" Adapters in ioBroker ist nötig. Dieser Adapter
+wird der für den Datenaustausch verwendet.
```

Comment:
Klingt komisch.

Reply:
Yeah, I recently updated the phrase. Still my preferred solution is a fully self contained package in the ioBroker adapter "store", I'll dig deeper how to get there after solving the diagram implementation. Will update this section then.